Tests/MapboxMapsTests/Location/Puck/Puck2DTests.swift

```diff
@@ -208,6 +209,22 @@ final class Puck2DTests: XCTestCase {
         return expectedProperties
     }
 
+    func makeExpectedSetLayerProperties(with location: InterpolatedLocation) -> [String: Any] {
+        let expectedLayoutLayerProperties = [LocationIndicatorLayer.LayoutCodingKeys: Any]()
+
+        var expectedPaintLayerProperties = [LocationIndicatorLayer.PaintCodingKeys: Any]()
+        expectedPaintLayerProperties[.location] = [location.coordinate.latitude, location.coordinate.longitude, location.altitude]
+        expectedPaintLayerProperties[.bearing] = 0
+
+        let expectedProperties = expectedLayoutLayerProperties
+            .mapKeys(\.rawValue)
+            .merging(
+                expectedPaintLayerProperties.mapKeys(\.rawValue),
+                uniquingKeysWith: { $1 })
+
+        return expectedProperties
+    }
```

Comment:
You can write this in-line more simply:

        let expectedProperties: [LocationIndicatorLayer.PaintCodingKeys: Any] = [
            .location: [
                location.coordinate.latitude,
                location.coordinate.longitude,
                location.altitude],
            .bearing: 0
        ]

And when you need to use it, to convert it to `[String: Any]`:

    expectedProperties.mapKeys(\.rawValue) as NSDictionary

There are some examples of this in the tests that have fastpath in the name